charts/nri-bundle/Chart.yaml

```diff
@@ -45,13 +45,21 @@ dependencies:
     version: 1.0.0
 
   - name: kube-state-metrics
-    version: 4.23.0
+    version: 4.25.0
     condition: ksm.enabled,kube-state-metrics.enabled
     repository: https://prometheus-community.github.io/helm-charts
+    alias: kube-state-metrics
+
+  - name: kube-state-metrics
```

Comment:
[discussed offline, adding it here for tracking porposes]

I still do not see clearly why adding a new dep is better (security or configuration wise) with respect to changing the image tag:
```
helm upgrade --install newrelic-bundle newrelic/nri-bundle 
 --set kube-state-metrics.image.tag = "v1.26.0" [...]
```